### PR TITLE
Fix the study progress breadcrumb styling

### PR DESCRIPTION
**Pull request by zarino**
The correct colour-to-colour triangle background images are
used based on the current and previous stages, and the entire
subnav background correctly colours its left and right ends
based on the state of the first and last stages.

Part of #39.

### CODE REVIEW

File: app/assets/stylesheets/_subnav.scss [hunk 1]
```diff
@@ -64,35 +64,60 @@
...
+    @include img-retina("stage-toffee-to-white.png", "stage-toffee-to-white@2x.png", 16px, 100px);
+    background-image: url("stage-toffee-to-white.svg"), none;
+  }
+
+  .step--done + .step--doing {
+    @include img-retina("stage-green-to-toffee.png", "stage-green-to-toffee@2x.png", 16px, 100px);
+    background-image: url("stage-green-to-toffee.svg"), none;
+  }
+
+  .step--done + .step--done {
```
Comment (zarino) [inline app/assets/stylesheets/_subnav.scss]: Nope. It's clearer when left separate.

File: app/assets/stylesheets/_subnav.scss [hunk 2]
```diff
@@ -64,35 +64,60 @@
...
+    @include img-retina("stage-white-to-white.png", "stage-white-to-white@2x.png", 16px, 100px);
+    background-image: url("stage-white-to-white.svg"), none;
+  }
+
+  .step--doing + .step--todo {
+    @include img-retina("stage-toffee-to-white.png", "stage-toffee-to-white@2x.png", 16px, 100px);
+    background-image: url("stage-toffee-to-white.svg"), none;
+  }
+
+  .step--done + .step--doing {
```
Comment (zarino) [inline app/assets/stylesheets/_subnav.scss]: Nope. It's clearer when left separate.

File: app/assets/stylesheets/_subnav.scss [hunk 3]
```diff
@@ -64,35 +64,60 @@
...
+  // We use Bootstrap's built-in img-retina() mixin to sort out
+  // retina bitmap images and sizes, and then override with an SVG
+  // if the browser supports it.
+
+  .step--todo + .step--todo {
+    @include img-retina("stage-white-to-white.png", "stage-white-to-white@2x.png", 16px, 100px);
+    background-image: url("stage-white-to-white.svg"), none;
+  }
+
+  .step--doing + .step--todo {
```
Comment (zarino) [inline app/assets/stylesheets/_subnav.scss]: Nope. It's clearer when left separate.

**Comment by zarino** [comment]
@stevenday could you give this a quick review and merge if happy?

**Comment by stevenday** [comment]
I've opened #82 to keep this up to date on the other work.

**Pull request closed without merging**